Habitica/src/main/java/com/habitrpg/android/habitica/helpers/FirstDayOfTheWeekHelper.java

```diff
@@ -8,34 +8,39 @@
 public class FirstDayOfTheWeekHelper {
 
     private int firstDayOfTheWeek;
+    private int dailyTaskFormOffset;
 
-    public FirstDayOfTheWeekHelper(String firstDayOfTheWeekSharedPref) {
+    private FirstDayOfTheWeekHelper(int dailyTaskFormOffset, int firstDayOfTheWeek) {
+        this.dailyTaskFormOffset = dailyTaskFormOffset;
+        this.firstDayOfTheWeek = firstDayOfTheWeek;
+    }
+
+    public static FirstDayOfTheWeekHelper newInstance(String firstDayOfTheWeekSharedPref) {
         switch (firstDayOfTheWeekSharedPref){
             case "1":
```

Comment:
Just noticing this now, but Why isn't this handled as an integer? Same with the other occurences. Seems Like it would be easier to use/store it as an integer.